application/classes/Ushahidi/Repository/Contact.php

```diff
@@ -124,4 +124,32 @@ public function getByContact($contact, $type)
 	{
 		return $this->getEntity($this->selectOne(compact('contact', 'type')));
 	}
+
+	// ContactRepository
+	public function getNotificationContacts($set_id = 0, $limit = false, $offset = 0)
+	{
+		$query = DB::select('contacts.id', 'contacts.type', 'contacts.contact')
+			->distinct(TRUE)
+			->from('contacts')
+			->join('notifications')
+			->on('contacts.user_id', '=', 'notifications.user_id')
+			->where('contacts.can_notify', '=', '1');
+
+		if ($set_id) {
```

Comment:
Right but its never used *without* the set_id .. could we just make that param required?